omnibus/config/software/cacerts.rb

```diff
@@ -25,33 +25,21 @@
 # doing this.
 name "cacerts"
 
-# We have a synthetic monitor on the latest cacerts file to warn us when the latest
-# cacerts bundle changes.
-# This allows us to always use up-to-date cacerts, without breaking all builds
-# when they change.
+# Omnibus breaks if there is no version on elements. You get an error like
+#   Software must specify a `version; to cache it in S3 (cacerts[/go/src/github.com/DataDog/datadog-agent/omnibus/config/software/cacerts.rb])!
 default_version "2025-08-12"
-source url: "https://curl.se/ca/cacert-#{version}.pem",
-       sha256: "64dfd5b1026700e0a0a324964749da9adc69ae5e51e899bf16ff47d6fd0e9a5e",
-       target_filename: "cacert.pem"
-
-relative_path "cacerts-#{version}"
 
 build do
   license "MPL-2.0"
   license_file "https://www.mozilla.org/media/MPL/2.0/index.815ca599c9df.txt"
 
   if windows?
-    mkdir "#{python_3_embedded}/ssl/certs"
-    copy "#{project_dir}/cacert.pem", "#{python_3_embedded}/ssl/certs/cacert.pem"
-    copy "#{project_dir}/cacert.pem", "#{python_3_embedded}/ssl/cert.pem"
+    mkdir "#{python_3_embedded}/embedded/ssl/cacerts"
+    copy "#{Omnibus::Config.project_root()}/../deps/cacerts/cacert.pem", "#{python_3_embedded}/embedded/ssl"
```

Comment:
We should probably be using the `source` directive instead, like we do for the [datadog-agent definition](https://github.com/DataDog/datadog-agent/blob/303fe6eb196422978696d628787840834743004a/omnibus/config/software/datadog-agent.rb#L31).

```ruby
source path: '../deps/cacerts'
```

And then this bit would become 
```suggestion
    copy "cacert.pem", "#{python_3_embedded}/embedded/ssl"
```

And the `license_file` above could also point to the local file that you vendored.

Reply:
Interesting.  I will try that. But I wonder about source having an option to exclude files. That seems to indicate a copy or symlink tree, rather than direct reference unpacked git repo.  I'll do some research.

Reply:
I don't want the extra copy, so I left it mostly as is. Just using the same cwd construct that you us in the run_as_root command.

Of course, CI may tell me I have to back it out.